src/app/space/wizard/space-wizard.module.ts

```diff
@@ -12,22 +13,16 @@ import { SpaceWizardComponent } from './space-wizard.component';
 
 @NgModule({
   imports: [
+    AddSpaceOverlayModule,
     CommonModule,
     FormsModule,
     Fabric8WitModule
   ],
   declarations: [
-    SpaceWizardComponent,
-    UniqueSpaceNameValidatorDirective,
-    ValidSpaceNameValidatorDirective,
-    SelectedItemsPipe,
-    VisibleItemsPipe,
-    TrustHtmlPipe,
-    TrustStylePipe
+    SpaceWizardComponent
   ],
   exports: [
-    SpaceWizardComponent,
-    UniqueSpaceNameValidatorDirective
```

Comment:
The omitted exports and declarations above should probably be restored. I haven't run through the old forge wizard fully, but suspect this will be required to continue working properly?

Reply:
restored